libs/@guardian/atoms-rendering/package.json

```diff
@@ -9,13 +9,13 @@
 		"@emotion/eslint-plugin": "11.11.0",
 		"@emotion/react": "11.11.1",
 		"@guardian/ab-core": "5.0.0",
-		"@guardian/commercial": "10.0.0",
-		"@guardian/consent-management-platform": "13.5.0",
-		"@guardian/eslint-plugin-source-foundations": "13.0.0",
-		"@guardian/eslint-plugin-source-react-components": "17.0.0",
-		"@guardian/libs": "15.1.0",
-		"@guardian/source-foundations": "12.0.0",
-		"@guardian/source-react-components": "15.0.1",
+		"@guardian/commercial": "10.8.0",
+		"@guardian/consent-management-platform": "13.6.1",
```

Comment:
Will defer to @guardian/commercial-dev @guardian/transparency-consent on these two.

Reply:
Thanks for updating "@guardian/consent-management-platform".